index.html

```diff
@@ -4,6 +4,19 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>cesartalvez! Software Engineering, Languages and Everthing in between!</title>
+
+  <style>
+
+    .badges-container {
+
+      display: flex;
+      flex-direction: row;
+    }
```

Comment:
It would be nice to center the columns in the screen; and also make sure it's mobile Friendly.